nilearn/datasets/utils.py

```diff
@@ -81,8 +84,10 @@ def _chunk_report_(bytes_so_far, total_size, initial_size, t0):
         If not resuming, set to zero.
     """
 
+    if cur_chunk_size == 0:
+        pass  # sys.stderr.write('\n')
     if not total_size:
-        sys.stderr.write("Downloaded %d of ? bytes\r" % (bytes_so_far))
+        sys.stderr.write("\rDownloaded %d of ? bytes" % (bytes_so_far))
```

Comment:
Putting the `\r` at the start allows us to keep the end output position, eliminating the need to print a newline later.